### ISSUE TITLE
508-defect-2 [SCREENREADER]: Headings not properly nested on health care page

### ISSUE THREAD

**Issue by AngelaFowler82** [department-of-veterans-affairs/va.gov-team#21035]
Label: 19375

# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Point of Contact: 
**VFS Point of Contact:** _First name only: Angela_

<h2>User Story or Problem Statement:</h2>
When navigating with a screen reader, I expect to be able to determine the outline of a page based on nested heading levels.
<h2>Details</h2>
On the main Health Care page, The heading "locations" is a level two heading, and the individual locations are level three headings. After the list of locations are two level three headings, "Manage your health online" and "In the spotlight." Since these mark a change in subject, they should be level two.
Acceptance Criteria

<h2>Environment</h2>
Consistent across all environments.
Steps to Recreate

Proposed Solution (if known)

WCAG or Vendor Guidance (optional)

Screenshots or Trace Logs
